### DISCUSSION TITLE
PM2 run strapi

### DISCUSSION THREAD

**Discussion by beactivesolutions** [strapi/strapi#6609] [Legacy Topics]
Hello,
how can i start the process through pm2 on Ubunut server.
Wher is server.js file?

**Comment by MattieBelt** [comment]
@beactivesolutions Did you take a look at [this guide](https://strapi.io/documentation/3.0.0-beta.x/guides/process-manager.html#process-manager)?

> So first let's create a `server.js` file that will let you run the pm2 command.

So you will need to create an new file in root `./server.js`


But you also can start it with the strapi commands: https://strapi.io/documentation/3.0.0-beta.x/guides/process-manager.html#starting-with-strapi-command
